openrarity/cli/opensea.py

```diff
@@ -46,11 +46,37 @@ def fetch_assets(
         help="Columns to print or write to file.",
     ),
 ):
+    """
+    If we provide token_ids data as a input, using OpenseaApi, it will collect assets data. And based on the 'rank' boolean flag it will calculate ranks. At the end, based on the 'write' flag it will write the data to either stdout or to a file.
+    If we didnt provide token_ids data as a input, it uses 'start_token_id' and 'end_token_id' as a range.Using OpenseaApi, it will collect assets data. And based on the 'rank' boolean flag it will calculate ranks. At the end, based on the 'write' flag it will write the data to either stdout or to a file.
```

Comment:
This should be a high level description of what the command is used for NOT a description of what the code is doing. Please reword. 

Expand on this but this is the idea...
> Given a list of token_ids or start and end ids for numbered collections, fetch the token metadata from Opensea. Optionally, the fetched tokens can be ranked by passing the `--rank` flag.

Reply:
fixed it.